src/GraphControl/Utils.h

```diff
@@ -0,0 +1,610 @@
+            return m_##n;                                                                                                                                      \
+        }                                                                                                                                                      \
+                                                                                                                                                               \
+    private:                                                                                                                                                   \
+        void set(t value)                                                                                                                                      \
+        {                                                                                                                                                      \
+            if (m_##n != value)                                                                                                                                \
+            {                                                                                                                                                  \
+                m_##n = value;                                                                                                                                 \
+                RaisePropertyChanged(L#n);                                                                                                                     \
+            }                                                                                                                                                  \
+        }                                                                                                                                                      \
+    }                                                                                                                                                          \
+                                                                                                                                                               \
+private:                                                                                                                                                       \
+    t m_##n;                                                                                                                                                   \
+                                                                                                                                                               \
+public:
+
+#define OBSERVABLE_PROPERTY_RW_ALWAYS_NOTIFY(t, n)                                                                                                             \
```

Comment:
Let's delete this from this file... I think we need to just remove this workaround now that we have access to TrimTrailingZeroes function after merging with master.

Reply:
removed!